querydsl-core/src/main/java/com/querydsl/core/DefaultQueryMetadata.java

```diff
@@ -59,7 +61,8 @@
 
     private List<OrderSpecifier<?>> orderBy = ImmutableList.of();
 
-    private List<Expression<?>> projection = ImmutableList.of();
+    @Nullable
```

Comment:
I moved the getter annotations now to the interface, but left the `@Nullable` annotations for the fields.